### DISCUSSION TITLE
Is it possible to use GoAccess on my own server to track other websites on other servers?

### DISCUSSION THREAD

**Discussion by JoshDebris** [allinurl/goaccess#2534] [Q&A]
Maybe a stupid question, but I really want to check out GoAccess. The problem is, that I do not host any websites on my own server. All my sites are on different servers from different providers. Is there a way to grab these Log-files - maybe with SSH or SFTP - and parse these loogfiles on my server?

Thanks in advance... :)

**Comment by allinurl** [comment]
If you prefer not to install it on your server, you have the option to run it remotely or directly from your local machine.

    # ssh -n root@server 'tail -f /var/log/apache2/access.log' | goaccess - --log-format=COMBINED

**Note**: SSH requires `-n` so GoAccess can read from stdin. Also, **make sure** to use SSH keys for authentication as it won't work if a passphrase is required.

**Reply by JoshDebris** [reply]
Ok., but what happens if I change the site?
Can GoAccess handle different Log Files?

**Reply by allinurl** [reply]
I'm not sure what you mean by "change the site." However, you can use the `-F` option with tail to specify that it should continue following the file if it changes. If you want to monitor multiple sites or files, you will need to SSH into each of them and use tail to track the logs individually. Afterward, you can pipe the data from each tail command to goaccess for analysis. e.g.,

```
ssh -n root@server1 'tail -f /var/log/apache2/access.log' & \
ssh -n root@server2 'tail -f /var/log/apache2/access.log' & \
ssh -n root@server3 'tail -f /var/log/apache2/access.log' | goaccess - --log-format=COMBINED
```
